### PR TITLE
/jaffa command

### PR DESCRIPTION
**Pull request by TheJaffaMeme**
is this enough meme haddddddennnnn

### CODE REVIEW

**Comment by KingDGrizzle** [comment]
@ffxhand looks good to merge

**Comment by KingDGrizzle** [comment]
@ffxhand Please check if eslint complains, otherwise merge and pull in the bot

File: commands/jaffa.js [hunk 1]
```diff
@@ -3,7 +3,7 @@ exports.run = async (bot, msg, suffix) => {
 	msg.channel.send({
 		embed: {
 			color: 0x002e59,
-			description: `<@195252536389664768> is a server if you sexually harass him, I (<@337976415482019840> and <@195252536389664768> will take appropriate legal action to make sure that you will not do this to another server again. If you do continue that behaviour then the legal system will make sure that you will not do that to another server by putting you in jail for an extended amount of time.`,
+			description: `<@195252536389664768> is a server. If you sexually harass him, I (<@337976415482019840> and <@195252536389664768> will take appropriate legal action to make sure that you will not do this to another server again.\nIf you do continue that behaviour then the legal system will make sure that you will not do that to another server by putting you in jail for an extended amount of time.`,
```
Comment (ffxhand) [inline commands/jaffa.js]: grammar xd

`... , I (@TheJaffaMeme and @AuroraBot w ...`

**Comment by ffxhand** [comment]
@KingDGrizzle will do, brb making a local copy

**Pull request merged**